appengine/js/lib-interface.js

```diff
@@ -65,13 +66,63 @@ BlocklyInterface.init = function() {
     linkButton.style.display = 'none';
   }
 
+  var loginDiv = document.getElementById('login');
+  if (loginDiv) {
+    BlocklyInterface.setupLogin();
```

Comment:
If we have the loginDiv here could we pass this into setupLogin and use this to add the loginButton to?

Reply:
This is actually leftover from previous version where I used loginDiv. Current logic does not use the div and instead adds the button the html page to right side of navbar (id "farSide"). I will update code to remove the old login divs